### PR TITLE
[13.0][MIG] website_breadcrumb

### PR DESCRIPTION
**Pull request by bjornbillen**

### CODE REVIEW

File: website_breadcrumb/views/layout.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+              inherit_id="website.layout"
+              customize_show="True"
+              name="Breadcrumbs">
+        <xpath expr="//header">
+            <t id="breadcrumb"
+               t-if="request.env['website.menu']
+                 .search([('url', '=', request.httprequest.path)], limit=1)">
+                <div class="container">
+                    <div class="row">
+                        <div class="col-md-12">
```
Comment (Tardo) [inline website_breadcrumb/views/layout.xml]:
```suggestion
                        <div class="col-12">
```

**Review by Tardo — CHANGES_REQUESTED** [review]
Looks Good. Code review

File: website_breadcrumb/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
+To configure the shown breadcrumbs, you need to:
+
+#. Enable developer mode.
+#. Go to Website Admin > Configuration > Settings > Menu > Configure website menus.
```
Comment (wytzejan) [inline website_breadcrumb/readme/CONFIGURE.rst]:
```suggestion
#. Go to Website Admin > Configuration > Menus.
```

**Review by wytzejan — COMMENTED** [review]
Nice. I did a functional review and found out that the readme was outdated.

**Review by claudiagn — COMMENTED** [review]
waiting for the runbot to work so I can review it functionally 👍

**Comment by acorpuz** [comment]
@bjornbillen Your changes look interesting, will this PR be merged?

**Review by Tardo — APPROVED** [review]
Looks good. Code review. Thanks!

**Comment by pedrobaeza** [comment]
I have rebase, squash administrative commits and apply the pre-commit commit before the real changes, so merging:

/ocabot merge nobump
/ocabot migration website_breadcrumb

**Pull request merged**